### PR TITLE
Add url param for office-addin-manifest to skip icon check in validation service

### PR DESCRIPTION
**Pull request by arttarawork**
Adds ?gates=DisableIconDimensionValidation to skip the localhost breaking change in our manifest validation service.

Bumps package version to 1.5.13

### CODE REVIEW

File: packages/office-addin-manifest/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "office-addin-manifest",
-  "version": "1.5.12",
```
Comment (TCourtneyOwen) [inline packages/office-addin-manifest/package.json:3]: We shouldn't bump this version manually - it gets bumped automatically during publishing to NPM

Comment (arttarawork) [reply packages/office-addin-manifest/package.json:3]: Gotcha, I'll change it back

File: packages/office-addin-manifest/package-lock.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "office-addin-manifest",
-  "version": "1.5.12",
+  "version": "1.5.13",
```
Comment (TCourtneyOwen) [inline packages/office-addin-manifest/package-lock.json:3]: Needs to be changed here as well

File: packages/office-addin-manifest/package.json (hunk 1 above)
Comment (arttarawork) [reply packages/office-addin-manifest/package.json]: Fixed in latest commit

File: packages/office-addin-manifest/package-lock.json (hunk 2 above)
Comment (arttarawork) [reply packages/office-addin-manifest/package-lock.json]: Fixed in latest commit

**Review by TCourtneyOwen — APPROVED** [review]

**Pull request merged**